packages/webapp/src/inductions/utils.ts

```diff
@@ -23,20 +27,28 @@ export const convertPendingProfileToMemberData = (
     };
 };
 
-export const getInductionStatus = (induction?: Induction) => {
-    if (!induction) return InductionStatus.invalid;
+export const getInductionStatus = (
+    induction?: Induction,
+    endorsements?: Endorsement[]
+) => {
+    if (!induction) return InductionStatus.Invalid;
 
     const isExpired = dayjs(eosBlockTimestampISO(induction.created_at))
         .add(INDUCTION_EXPIRATION_DAYS, "day")
         .isBefore(dayjs());
 
-    return isExpired
-        ? InductionStatus.expired
-        : !induction.new_member_profile.name
-        ? InductionStatus.waitingForProfile
+    if (isExpired) return InductionStatus.Expired;
+
+    const isWaitingForDonation =
+        endorsements?.every((e) => e.endorsed === 1) ?? false;
```

Comment:
Could be simplified by: `Boolean(endorsements?.every((e) => e.endorsed === 1))`

Reply:
Yeah, either works. I'll wrap with `Boolean`. 👍